client/actions/eventsPlanning/ui.js

```diff
@@ -17,6 +17,7 @@ import {showModal} from '../index';
 const fetch = (params = {}) => (
     (dispatch, getState, {$location, $timeout}) => {
         dispatch(self.requestEventsPlanning(params));
+        dispatch(eventsApi.fetchEventTemplates());
```

Comment:
Initial fetching of items such as EventTemplates should be placed in the parent controller (https://github.com/superdesk/superdesk-planning/blob/master/client/controllers/PlanningController.js#L74).

Websocket notifications will take over from that point making sure the templates are up to date.